test/metabase/test/util.clj

```diff
@@ -743,6 +744,39 @@
   [collection-or-id & body]
   `(do-with-discarded-collections-perms-changes ~collection-or-id (fn [] ~@body)))
 
+(declare with-discard-model-updates)
```

Comment:
introduced in https://github.com/metabase/metabase/commit/3d2d69b687f5658d07a74cc920b727f13e093bb4, and this macro is used as part of this PR